buildSrc/version.properties

```diff
@@ -38,6 +38,8 @@ commonscodec      = 1.13
 hamcrest          = 2.1
 securemock        = 1.2
 mocksocket        = 1.2
+mockito           = 1.9.5
```

Comment:
Any side effects of this? Are these the same version that was being brought in my mocksocket/mockito?

Reply:
Yes, this is exactly the same version we use in build tools: https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/build.gradle#L126.

Reply:
... and https://github.com/elastic/securemock/blob/master/pom.xml#L41 (securemock)